account_statement_operation_rule_dunning_fees/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Bank Statement Operation Rules with Dunning Fees
```

Comment:
Isn't this called *_fees in general without the word dunning as it serves the same for any kind of fee?